pages/appbuilder/AppBuilderPage.tsx

```diff
@@ -122,6 +123,20 @@ interface Props extends IAppBuilderSettingsSession {
 	example?: string;
 }
 
+/**
+ * Create rendering hints for the container.
+ * @param container 
+ * @returns 
+ */
+const createContainerHints = (container: IAppBuilderContainer) : IAppBuilderTemplatePageContainerHints | undefined => {
+	// if the bottom container contains tabs, prefer vertical layout
+	if (container.name === "bottom" && container.tabs && container.tabs.length > 0) {
+		return {
+			preferVertical: true
+		};
+	}
+};
+
```

Comment:
@mathieu-shapediver this summarizes how I solved the bottom container layout issue: https://shapediver.atlassian.net/browse/SS-7859

Reply:
@mathieu-shapediver we can use the same approach to create container-content-specific rendering hints for other cases, e.g. the top container